### PR TITLE
Issue 314 int control param android causes invalid opg property

### PR DESCRIPTION
**Pull request by surabhi-sequoiaat**
Fixing issue : Integer values are read as double by default.

### CODE REVIEW

File: android/src/main/java/io/branch/rnbranch/RNBranchModule.java [hunk 1]
```diff
@@ -1013,7 +1013,12 @@ private static Object getReadableMapObjectForKey(ReadableMap readableMap, String
             case Boolean:
                 return readableMap.getBoolean(key);
             case Number:
-                return readableMap.getDouble(key);
+                if(readableMap.getDouble(key)%1==0){
```
Comment (sojanpr) [inline android/src/main/java/io/branch/rnbranch/RNBranchModule.java]: Please format the code with spacings

**Comment by sojanpr** [comment]
@surabhi-sequoiaat Not sure what is the issue this PR is addressing. The values added to link params are converted to Strings before adding them. Are we seeing any issue when double values are added in the link param value?

**Comment by sequoiaat** [comment]
@sojanpr The problem was that the integer values are getting converted to double before converting it in to String. Now we have changed it so that integer will get integer string and double will continue getting double string.

File: android/src/main/java/io/branch/rnbranch/RNBranchModule.java (hunk 1 above)
Comment (sequoiaat) [reply android/src/main/java/io/branch/rnbranch/RNBranchModule.java]: i did the formatting, please review

**Comment by sojanpr** [comment]
LGTM 👍

**Pull request merged**